src/ast2ram/AstToRamTranslator.cpp

```diff
@@ -200,28 +200,20 @@ std::vector<std::map<std::string, std::string>> AstToRamTranslator::getOutputDir
     return outputDirectives;
 }
 
-Own<ram::RelationReference> AstToRamTranslator::createRelationReference(const std::string name) {
-    auto it = ramRels.find(name);
-    assert(it != ramRels.end() && "relation name not found");
-
-    const ram::Relation* relation = it->second.get();
-    return mk<ram::RelationReference>(relation);
-}
-
-Own<ram::RelationReference> AstToRamTranslator::translateRelation(const ast::Atom* atom) {
-    return createRelationReference(getRelationName(atom->getQualifiedName()));
+std::string AstToRamTranslator::translateRelation(const ast::Atom* atom) {
+    return getRelationName(atom->getQualifiedName());
```

Comment:
It looks like `getRelationName(const ast::QualifiedName& id)' could instead be `getRelationName(const ast::Atom& atom)`, which would simplify the code, and perhaps avoid the need for this particular function entirely.

Since we're no longer translating a relation, perhaps rename these to `translateRelationName`.

Reply:
I would like to wait for this refactoring: we will need to revisit the translation holistically. @azreika made good progress with refactoring the Clause translation.

Reply:
Agree, let's wait for another pass before cleaning this up so changes are more stable.